classes/controllers/genre_controller.rb

```diff
@@ -0,0 +1,41 @@
+require_relative './provider'
+require_relative '../models/genre'
+
```

Comment:
Hi @akhror-valiev  ✋
Good job so far!

Highlights!
Followed Gitflow ✔️
Linters are passing ✔️
Descriptive Pull request ✔️
Professional readme ✔️
Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

Status: Required Changes ♻️⛔️
Check the comments under the review.

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me @1GORDON if something is not 100% clear.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

- [ ] Please add sql file for creating the database of those tables that you implemented

Reply:
Thanks, @ZuhRa-HashiMi, totally forget to add it 👍